tripal_chado/includes/TripalImporter/GFF3Importer.inc

```diff
@@ -1881,6 +1944,13 @@ class GFF3Importer extends TripalImporter {
               }
               if ($matched_type_id == $f->type_id and $matched_organism_id == $f->organism_id) {
                 $this->features[$f->uniquename]['feature_id'] = $f->feature_id;
+                $this->features[$f->uniquename]['name'] = $f->name;
+		# SOFIA: check to see if the name has changed
+		if ($f->name != $matched_feature['name']) {
+                  ## need to update name of feature
+		  ## add flag to cached feature that indicates updated needed
+		  $this->update_names[$f->feature_id]=$matched_feature['name'];
+	        }
```

Comment:
indentation, and change comments to forwards slashes

Reply:
Done.